tests/test_landing_pad_tracking.py

```diff
@@ -0,0 +1,389 @@
+        detections1: "list[object_in_world.ObjectInWorld]"):
+        """
+        Test if marking false positive adds detection to list of false positives
+        """
+        false_positive1 = object_in_world.ObjectInWorld.create(0, 0, 1)[1]
+        false_positive2 = object_in_world.ObjectInWorld.create(2, 2, 1)[1]
+        tracker._LandingPadTracking__unconfirmed_positives = detections1
+        expected = [false_positive1, false_positive2]
+        expected_unconfirmed_positives = [detections1[2], detections1[3], detections1[4]]
+        
+        tracker.mark_false_positive(false_positive1)
+        tracker.mark_false_positive(false_positive2)
+
+        assert tracker._LandingPadTracking__false_positives == expected
+        assert tracker._LandingPadTracking__unconfirmed_positives == expected_unconfirmed_positives
+
+    # pylint: enable=protected-access
+
+
+class TestMarkConfirmedPositive:
+    """
+    Test if landing pad tracking correctly marks a detection as a confirmed positive
+    """
+    # Required for testing
+    # pylint: disable=protected-access
+    def test_mark_confirmed_positive(self, tracker: landing_pad_tracking.LandingPadTracking):
+        """
+        Test if marking confirmed positive adds detection to list of confirmed positives
+        """
+        confirmed_positive = object_in_world.ObjectInWorld.create(1, 1, 1)[1]
+        expected = [confirmed_positive]
+        
+        tracker.mark_confirmed_positive(confirmed_positive)
+
+        assert tracker._LandingPadTracking__confirmed_positives == expected
+    
+    def test_mark_multiple_confirmed_positives(self,
+                                               tracker: landing_pad_tracking.LandingPadTracking):
+        """
+        Test if marking confirmed positive adds detection to list of confirmed positives
+        """
+        confirmed_positive1 = object_in_world.ObjectInWorld.create(1, 1, 1)[1]
+        confirmed_positive2 = object_in_world.ObjectInWorld.create(2, 2, 1)[1]
+        expected = [confirmed_positive1, confirmed_positive2]
+        
+        tracker.mark_confirmed_positive(confirmed_positive1)
+        tracker.mark_confirmed_positive(confirmed_positive2)
+
+        assert tracker._LandingPadTracking__confirmed_positives == expected
+
+    # pylint: enable=protected-access
+
+
+class TestLandingPadTracking:
+    """
+    Test landing pad tracking run function
+    """
+    # Required for testing
+    # pylint: disable=protected-access
+    def test_run_with_empty_detections_list(self, tracker: landing_pad_tracking.LandingPadTracking):
+        """
+        Test run method with empty detections list
+        """
+        result, actual = tracker.run([])
+        assert not result
+        assert actual is None
+
+    def test_run_one_input(self,
+                              tracker: landing_pad_tracking.LandingPadTracking,
+                              detections1: "list[object_in_world.ObjectInWorld]"):
```

Comment:
Fix the indentation so that `self` , `tracker` , and `detections1` are lined up vertically.